benchmarks/dali_vs_python/pipelines_compare_client.py

```diff
@@ -182,67 +166,70 @@ def generate_outputs(output_name):
     return [tritonclient.grpc.InferRequestedOutput(output_name)]
 
 
-def infer_dali(triton_client, batch):
+def infer_dali(triton_client, batch, model_name_prefix):
+    model_name = model_name_prefix + "_dali"
+    triton_client.load_model(model_name)
+
     inputs = generate_inputs("DALI_INPUT_0", batch.shape, "UINT8")
     outputs = generate_outputs("DALI_OUTPUT_0")
 
     # Initialize the data
     inputs[0].set_data_from_numpy(batch)
 
     # Test with outputs
-    results = triton_client.infer(model_name="rn50_dali", inputs=inputs, outputs=outputs)
+    results = triton_client.infer(model_name=model_name, inputs=inputs, outputs=outputs)
 
     # Get the output arrays from the results
     output0_data = results.as_numpy("DALI_OUTPUT_0")
+
+    triton_client.unload_model(model_name)
     return output0_data
 
 
-def infer_python(triton_client, batch):
+def infer_python(triton_client, batch, model_name_prefix):
+    model_name = model_name_prefix + "_python"
+    triton_client.load_model(model_name)
+
     inputs = generate_inputs("PYTHON_INPUT_0", batch.shape, "UINT8")
     outputs = generate_outputs("PYTHON_OUTPUT_0")
 
     # Initialize the data
     inputs[0].set_data_from_numpy(batch)
 
     # Test with outputs
-    results = triton_client.infer(model_name="rn50_python", inputs=inputs, outputs=outputs)
+    results = triton_client.infer(model_name=model_name, inputs=inputs, outputs=outputs)
 
     # Get the output arrays from the results
     output0_data = results.as_numpy("PYTHON_OUTPUT_0")
+
+    triton_client.unload_model(model_name)
     return output0_data
 
 
-def main():
-    FLAGS = parse_args()
+def main(model_name):
     try:
         triton_client = tritonclient.grpc.InferenceServerClient(url=FLAGS.url,
                                                                 verbose=FLAGS.verbose)
     except Exception as e:
         print("channel creation failed: " + str(e))
         sys.exit()
 
-    print("Loading images")
+    print("Loading samples")
 
-    if FLAGS.img_dir:
-        image_data = load_images(FLAGS.img_dir, max_images=FLAGS.batch_size * FLAGS.n_iter)
-    elif FLAGS.img:
-        image_data = [load_image(FLAGS.img)]
-    else:
-        print("No image specified")
-        sys.exit(1)
+    image_data = [load_sample(FLAGS.img)]
```

Comment:
```suggestion
    image_data = [load_sample(FLAGS.img)]
```
img? Maybe we should rename that too?

Reply:
Renamed. Also renamed some other occurrences.